config/clusterapi/controlplane/bases/controlplane.cluster.x-k8s.io_k0scontrolplanetemplates.yaml

```diff
@@ -0,0 +1,225 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.11.4
```

Comment:
Btw, isn't this a tad bit old? Maybe it's on purpose.

Reply:
Yeah, maybe. But it's the version, that we use: https://github.com/k0sproject/k0smotron/blob/main/Makefile.variables#L2

I think we can update it in a separate PR